lib/src/combine_latest.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+/// Combine the latest value from the source stream with the latest value from
+/// [combineWith] using [combine].
+///
+/// No event will be emitted from the result stream until both the source stream
+/// and [combineWith] have each emitted at least one event. The result stream
```

Comment:
I would separate these all out into their own paragraphs (or bullets?) - they are all descriptions of distinct behaviors.

Reply:
Done